tests/test_factory.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/python
+import socket_ipc_factory, endpoints, threading, time, unittest
+
+class TestIPCFactory(unittest.TestCase):
+  
+  def test_radio_belief(self):
```

Comment:
Even for unit tests, I generally put docstrings explaining what each suite and each individual test is for. It actually prints them out when you run the tests, which is helpful sometimes.